doc/man1/flux-dmesg.adoc

```diff
@@ -1,5 +1,5 @@
 // flux-help-description: manipulate broker log ring buffer
-FLUX-GETATTR(1)
+FLUX-DMESG(1)
 ===============
```

Comment:
Some asciidoc renderers complain and error out when the number of equal signs below the title don't match the title length (I know, Iknow) (Thanks @garlick for pointing this out)

While asciidoctor doesn't complain, and builds the manpage, it seems to result in a very poorly formatted manpage so this should be fixed:
e.g. right now the page is rendered as:
```
FLUX-DMESG(1)               Flux Command Reference               FLUX-DMESG(1)

FLUX-DMESG(1)

           NAME

               flux-dmesg - access broker ring buffer

               SYNOPSIS
               --------
               *flux* *dmesg* ['OPTIONS']
```
vs
```
FLUX-DMESG(1)               Flux Command Reference               FLUX-DMESG(1)

NAME
       flux-dmesg - access broker ring buffer

SYNOPSIS
       flux dmesg [OPTIONS]


```

Reply:
@grondo:  Just seeing this now (thanks, @gonsie).  I've fixed the underscoring.  Taking a look at the commit message truncation now.